### ISSUE TITLE
Could not create connection

### ISSUE THREAD

**Issue by IceBlueFire** [iConomy/Core#76]
Using iConomy 5.0, CB 740.
(Other mods in use: Essentials, iColand, Mob Bounty, MultiVerse, MultiInv, Citizens)

I wasn't doing anything (literally, idling while doing other things on the computer), when suddenly the console was getting these errors:

19:46:00 [INFO] [iConomy] Could not create connection: java.sql.SQLException: Lo
gin timeout
19:46:31 [INFO] [iConomy] Could not create connection: java.sql.SQLException: Lo
gin timeout
19:47:02 [INFO] [iConomy] Could not create connection: java.sql.SQLException: Lo
gin timeout
19:47:33 [INFO] [iConomy] Could not create connection: java.sql.SQLException: Lo
gin timeout
19:56:00 [INFO] [iConomy] Could not create connection: java.sql.SQLException: Lo
gin timeout
19:56:00 [SEVERE] Exception in thread "Timer-0"
19:56:00 [SEVERE] java.lang.NullPointerException
19:56:00 [SEVERE]       at com.iConomy.system.Interest.run(Interest.java:112)
19:56:00 [SEVERE]       at java.util.TimerThread.mainLoop(Unknown Source)
19:56:00 [SEVERE]       at java.util.TimerThread.run(Unknown Source)

After that, it lags me out of the server and i can't log back in until i restart the server.

**Comment by nijikokun** [comment]
Server is timing out when doing the transactions from interest. Lots of people logged in?

**Comment by IceBlueFire** [comment]
No, it was literally just me on the server, and like i mentioned i wasn't even playing.  I was just idling on the server.

**Comment by nijikokun** [comment]
Check your database settings

**Comment by IceBlueFire** [comment]
What files would i need to remove/repair in order to fix it?

**Comment by nijikokun** [comment]
It's a database issue, not a plugin issue.

**Comment by IceBlueFire** [comment]
No i know.  But i'm saying would i have to delete the h2 file in the libs folder to fix it? 

I'm using the h2 database btw.

**Comment by Tearlow** [comment]
I would have to report the very same thing. However, I did go ahead and made a copy of my database and deleted to start out from scratch again. But this time on launch It created a new database with the size of 2.3mb~. After looking into the file I can say that it is now utterly corrupted instead, parts of the database is there but filled with illegal characters. I deleted everything and restarted (Config etc.) to let the plugin create that as well. Same thing, Config/message file was created properly though. But the database was once more corrupted. Putting back the backed up database reverts this back to square one- Could not create connection.
